handsontable/src/plugins/autoRowSize/autoRowSize.js

```diff
@@ -66,6 +67,33 @@ const ROW_WIDTHS_MAP_NAME = 'autoRowSize';
  *   // code...
  * }
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * const hotRef = useRef();
+ *
+ * ...
+ *
+ * // First, let's contruct Handsontable
+ * <HotTable
+ *   ref={hotRef}
+ *   data={getData()}
+ *   autoRowSize={true}
+ * />
+ *
+ * ...
+ *
+ * // Access to plugin instance:
+ * const plugin = hotRef.getPlugin('autoRowSize');
```

Comment:
Same as above - you'll need to call the API methods from the `.current.hotInstance` property of the referenced component.

Reply:
Changed within: [`8464a8e` (#9791)](https://github.com/handsontable/handsontable/pull/9791/commits/8464a8efd8e87971f4d593025622e98b0e8b20ae)